action.yml

```diff
@@ -10,6 +10,10 @@ inputs:
     description: 'Output filename'
     default: ./bom.xml
     required: false
+  format:
+    description: 'Output format'
+    default: xml
+    required: false
 runs:
-  using: 'node12'
+  using: 'node16'
```

Comment:
please revert. and make an extra PR for this.

i need to read wether bumping node here is to be treated as a breaking change.
and this may take a while. and might cause your feature to get stuck in PR-limbo.


so please keep your PR to one feature/change, so it can be merged and released fast.

Reply:
yepp. bumping node is expected to be a breaking change. 

compare:
- https://github.com/actions/checkout/releases/tag/v3.0.0
- https://github.com/actions/checkout/releases/tag/v4.0.0

Reply:
caused #9